src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -437,6 +449,121 @@ public static void WriteNumberScientificAsProperty(bool indented)
                 "{\"m\\u00eal\\u00e9e\":1e6}");
         }
 
+        [Fact]
+        public static void WriteValueUnicodeEscapeSequence()
+        {
+            string unicodeString = "\u0421\u0430\u0439\u043D \u0443\u0443? \u04E9\u04E9";
+            string json = $"[\"{unicodeString}\"]";
+            var buffer = new ArrayBufferWriter<byte>(1024);
+            ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(Encoding.UTF8.GetBytes(json), 1);
+            string expectedStr = GetEscapedExpectedString(unicodeString, StringEscapeHandling.EscapeNonAscii);
+
+            using (JsonDocument doc = JsonDocument.Parse(sequence, s_options))
+            {
+                JsonElement target = doc.RootElement[0];
+
+                using (var writer = new Utf8JsonWriter(buffer))
+                {
+                    target.WriteValue(writer);
+                    writer.Flush();
+                }
+                string str = Encoding.UTF8.GetString(buffer.WrittenSpan);
```

Comment:
You don't use this variable, and this line is making the netfx build fail:

```
JsonElementWriteTests.cs(470,54): error CS1503: Argument 1: cannot convert from 'System.ReadOnlySpan<byte>' to 'byte[]' [D:\a\1\s\src\System.Text.Json\tests\System.Text.Json.Tests.csproj]
##[error]JsonElementWriteTests.cs(470,54): error CS1503: (NETCORE_ENGINEERING_TELEMETRY=Build) Argument 1: cannot convert from 'System.ReadOnlySpan<byte>' to 'byte[]'
```

Seems like the line can just be deleted.

Reply:
uh, forgot to remove